mlos_core/mlos_core/optimizers/bayesian_optimizers/smac_optimizer.py

```diff
@@ -350,8 +351,10 @@ def _suggest(
             warn(f"Not Implemented: Ignoring context {list(context.columns)}", UserWarning)
 
         trial: TrialInfo = self.base_optimizer.ask()
-        trial.config.is_valid_configuration()
-        self.optimizer_parameter_space.check_configuration(trial.config)
+        trial.config.check_valid_configuration()
+        ConfigSpace.Configuration(
+            self.optimizer_parameter_space, values=trial.config
```

Comment:
```suggestion
            self.optimizer_parameter_space, values=trial.config,
```

(i.e., add a trailing comma and let `black` reformat to split the args across lines - my general preference is to either have everything on one line, or else let it do one line per arg)